### PR TITLE
Receiving translation for OIE

### PR DESCRIPTION
**Pull request by eng-prod-CI-bot-okta**
OIE translation for the latest commit: https://github.com/okta/okta-help/commits/docs_translations_OIE

### CODE REVIEW

File: oie/ja-jp/content/resources/scripts/js/ignore-dompurify.js [hunk 1]
```diff
@@ -13,17 +13,3 @@ requirejs.onError = function (err) {
   }
   throw err;
 };
```
Comment (mauriciocastillosilva-okta) [inline oie/ja-jp/content/resources/scripts/js/ignore-dompurify.js:15]: What is the status for lower case redirects? I see we are removing the function, so just want to know why we don't need it anymore

File: oie/ja-jp/content/topics/dashboard/dashboard.htm [hunk 2]
```diff
@@ -124,7 +124,7 @@
                                 </div>
                             </div>
                             <div class="title-bar-layout outer-row">
-                                <div class="logo-wrapper"><a class="logo" href="https://support.okta.com/help/s/?language=ja" alt="Okta"></a>
+                                <div class="logo-wrapper"><a class="logo" href="https://help.okta.com/okta_help.htm?id=csh-index" alt="Okta"></a>
```
Comment (mauriciocastillosilva-okta) [inline oie/ja-jp/content/topics/dashboard/dashboard.htm:127]: If I navigate directly to https://help.okta.com/okta_help.htm?id=csh-index I always get the content in Engish, even if I had selected Japanese in the language switcher earlier. Do we need some url param to indicate the language?

**Review by mauriciocastillosilva-okta — COMMENTED** [review]
Just a couple of questions, but looks good overall. (Impossible to review every single file, so I looked at just a few of them)

File: oie/ja-jp/content/resources/scripts/js/ignore-dompurify.js (hunk 1 above)
Comment (emikhasyak-okta) [reply oie/ja-jp/content/resources/scripts/js/ignore-dompurify.js:15]: we actually need it, I'll update LL branch

**Pull request closed without merging**